README.md

```diff
@@ -68,6 +69,21 @@ $ sudo find / -name qmake
 ## Run:
 ``` ./start.sh [options] [script file name] ```
 
+## Documentation generation
+### Download and install [ldoc](stevedonovan.github.io/ldoc/manual/doc.md.html)
+```
+sudo apt install luarocks
+sudo luarocks install penlight
```

Comment:
@VVeremjova,
Please, try to install `luasec` first
```sudo luarocks install luasec```
If it helps, I'll update an instruction.